packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -421,7 +419,7 @@ export const getLinksSupplier =
 
       if (href) {
         href = locationUtil.assureBaseUrl(href.replace(/\n/g, ''));
-        href = replaceVariables(href, dataLinkScopedVars, VariableFormatID.PercentEncode);
+        href = replaceVariables(href, dataLinkScopedVars, 'percentencodeasuri');
```

Comment:
rather than the string... and since this is against main, we can define a new VariableFormatID from:

https://github.com/grafana/grafana/blob/main/packages/grafana-schema/src/common/variables.cue#L23

I think `uriencode` makes the most sense